runelite-client/src/main/java/net/runelite/client/util/OSXUtil.java

```diff
@@ -40,7 +40,7 @@
 	 *
 	 * @param gui The gui to enable the fullscreen on.
 	 */
-	public static void tryEnableFullscreen(ClientUI gui)
+	public static void tryEnableFullscreen(JFrame gui)
```

Comment:
Should this be in SwingUtil?

Reply:
No, because this is OSX-specific functionality